lib/actions/api.js

```diff
@@ -188,6 +185,15 @@ export function carRentalQuery (params) {
   return createQueryAction('car_rental', carRentalResponse, carRentalError)
 }
 
+// Vehicle rental (e.g. Lime eScooter) locations lookup query
+
+export const vehicleRentalResponse = createAction('VEHICLE_RENTAL_RESPONSE')
+export const vehicleRentalError = createAction('VEHICLE_RENTAL_ERROR')
```

Comment:
"Vehicle rental" feels poorly named. We have very mode-specific names for every other rental type (e.g., bike, car). I'm guessing this name is aimed at a future refactor where everything is a vehicle so the generic name makes sense. If this is the case, it deserves a comment somewhere or we should honestly just name the thing scooter rental -- otherwise it could get confusing.

Reply:
Yeah, it's intended to be for a future refactor. I'll add some comments. Also, if you have any thoughts on combining the APIs in a future refactor, that is appreciated.